extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -275,3 +290,30 @@ export function getSqlNumber(value: azdata.DbCellValue): number {
 export function getSqlBoolean(value: azdata.DbCellValue): boolean {
 	return value.isNull ? false : value.displayValue === '1';
 }
+
+export async function getDatabasesList(sourceConnectionId: string): Promise<azdata.DatabaseInfo[]> {
+	const ownerUri = await azdata.connection.getUriForConnection(sourceConnectionId);
+	const queryProvider = azdata.dataprotocol.getProvider<azdata.QueryProvider>(
+		'MSSQL',
```

Comment:
is it possible to avoid hardcoding the MSSQL value? 

e.g. look at the old implementation in `databaseSelectorPage.ts`:
![image](https://user-images.githubusercontent.com/11844501/198162747-75d335e2-22e5-4450-bf76-888fbcb90d84.png)

Even though it's not exactly the same (we are getting a metadata provider instead of a query provider), we were able to get the provider info from the connection... could we do the same thing here?

Reply:
updated to use the providerId from the connection.